plugins/operators/group_by.py

```
@@ -0,0 +1,200 @@
+"""
```

Pylint (3.3.7)

[error] Unrecognized option found: optimize-ast, files-output, function-name-hint, variable-name-hint, const-name-hint, attr-name-hint, argument-name-hint, class-attribute-name-hint, inlinevar-name-hint, class-name-hint, module-name-hint, method-name-hint, no-space-check

(E0015)